app/jobs/application_job.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class ApplicationJob < ActiveJob::Base
```

Comment:
I was curious what this new empty class is for given that there is already GenericJob.

Reply:
ApplicationJob is a modern rails convention. If it is missing it is generated when you do `bin/rails generate job MyNewJobName`